### PR TITLE
DragNDrop fix

### PR DESCRIPTION
**Pull request by soslow4u**

### CODE REVIEW

**Comment by mattroks101** [comment]
Can unconscious people drag and drop?

**Comment by LethalGhost** [comment]
Agree with @mattroks101 
@soslow4u 
Use `MOB.incapacitated()` or `MOB.is_physically_disabled()` instead direct `stat` check

File: code/_onclick/drag_drop.dm [hunk 1]
```diff
@@ -10,6 +10,8 @@
 		return
 	if(!Adjacent(usr) || !over.Adjacent(usr)) // should stop you from dragging through windows
 		return
+	if(usr.stat == DEAD || usr.incapacitated())
```
Comment (LethalGhost) [inline code/_onclick/drag_drop.dm]:
```suggestion
	if(usr.incapacitated())
```

**Comment by ACCount12** [comment]
What exactly does this PR disable? Just the table interactions?

**Comment by LethalGhost** [comment]
@ACCount12 
Bug what allowed ghosts to move items on table

**Comment by ACCount12** [comment]
That was a feature.

**Comment by mattroks101** [comment]
Ghosts moving items on tables was an intended feature?

**Comment by ACCount12** [comment]
Yep, I remember a discussion about it. It was decided that ghosts shuffling items is basically the same thing as ghosts spinning chairs.

**Pull request closed without merging**